### DISCUSSION TITLE
mpm --update depreciated, what is command line alternative?

### DISCUSSION THREAD

**Discussion by SinOfNature** [MiKTeX/miktex#1346] [General]
Hi,

I need a **command line** way of updating the packages (not using admin console).  I was using the following commands on Windows 10:

"C:\Program Files\MiKTeX\miktex\bin\x64\mpm.exe" --admin --find-updates   
"C:\Program Files\MiKTeX\miktex\bin\x64\mpm.exe" --admin --update 

I'm getting a warning that these commands are depreciated and will be removed in later versions.  I also noticed that the command mpm --help does even show those options.

What is the correct way of doing this that does not use the GUI console.  I need to do this via script.  All the documentation seems to keep pointing me to the Admin Console (GUI).  

Any help appreciated.

Iain

**Comment by koppor** [comment]
Go for the `miktex packages` command:

```
miktex --admin packages update-package-database
miktex --admin packages update
miktex packages update
```

See https://docs.miktex.org/manual/miktex-packages.html for more information
